product-configuration-extension/shopify.ui.extension.toml.liquid

```diff
@@ -0,0 +1,16 @@
+name = "{{ name }}"
```

Comment:
Can you update the TOML to much the new Unified format? The file name needs to be `shopify.extension.toml.liquid` and it should follow the same format as this one: https://github.com/Shopify/extensions-templates/blob/main/block-extension/shopify.extension.toml.liquid